dev-manuals/test-plans/lecturer-interface-frontend/plans/u-lecturer-interface-13-edit-minigame-configuration-chickenshock.md

```diff
@@ -0,0 +1,54 @@
+# Lecturer-Interface: Edit minigame configuration of chickenshock (`u.lecturer-interface-13`)
+
+
+Version: V1.0, 09.08.2022 \
+Author: Max Kästner
+
+## Description
+
+The chickenshock configuration of a minigame can be edited in a modal. For minigames of a world the path is `/courses/<course-id>/worlds/<world-index>/minigames` and for a dungeon `/courses/<course-id>/worlds/<world-index>/dungeons/<dungeon-index>/minigames`.
+
+## Precondition
+
+At least one course is created. The tester is on the view of an area minigames list as described in the description above.
```

Comment:
```suggestion
At least one course with a `CHICKENSHOCK` minigame is created. The tester is located where the `chickenshock` instance can be configured.
```

Reply:
Why should this be as precondition that the minigame is already a chickenshock game?

Reply:
See one below, because you then delete the first four steps in the message below, and because this setup has nothing to do with the test itself.